server/events/pull_closed_executor.go

```diff
@@ -73,6 +75,24 @@ var pullClosedTemplate = template.Must(template.New("").Parse(
 
 // CleanUpPull cleans up after a closed pull request.
 func (p *PullClosedExecutor) CleanUpPull(repo models.Repo, pull models.PullRequest) error {
+	pullStatus, err := p.DB.GetPullStatus(pull)
+	if err != nil {
+		// Log and continue to clean up other resources.
+		p.Logger.Err("retrieving pull status: %s", err)
+	}
+
+	if pullStatus != nil {
+		for _, project := range pullStatus.Projects {
+			// TODO: Set projectName to "<dir>/<workspace>" when project name is not set.
```

Comment:
Mind creating a jira for this and then putting the ticket number here? will make it easier to track the places we need to support this.

Reply:
Done!